README.md

```diff
@@ -101,13 +101,48 @@ mint = mintapi.Mint(..., driver=driver)
 
 ### MFA Authentication Methods
 
-As of v2.0, `mfa_method` is only required if your login flow presents you with the option to select which Multifactor Authentication Method you wish to use, typically as a result of your account configured to accept different methods.  
-
-If `mintapi` detects that your Mint account uses IMAP and your email host provides IMAP access, you can specify your IMAP login details.  This will automate the retrieval of the MFA code from your email and entering it into Mint.  If you use IMAP in conjunction with `keyring`, then you can store your IMAP password (`imap-password`) in keyring.  To do so, simply omit `imap-password` and you will initially be prompted for the password associated with your IMAP account.  Then, on subsequent uses of your IMAP account, you will not have to specify your password.
-
-If `mfa-method` is soft-token then you must also pass your `mfa-token`. The `mfa-token` can be obtained by going to [your mint.com settings](https://mint.intuit.com/settings.event?filter=all) and clicking on 'Intuit Account'. From there go to *Sign In & Security* -> *Two-step verification*. From there, enable the top option however you wish (either text or email is fine). After that, start the process to enable the *Authenticator app* option and when you get the part where you see the QR code, **copy the manual setup code** that appears next to it. Careful where you store this as it allows anyone to generate TOTP codes. This is the token that you will pass to `mfa-token` in either the python api or from the command line.
-
-While Mint supports authentication via Voice, `mintapi` does not currently support this option.  Compatability with this method will be added in a later version.
+You can handle MFA in one of two ways: email or TOTP
+(**T**ime-based **O**ne-**T**ime **P**assword).
+TOTP is strongly recommended.
+
+While Mint supports authentication via Voice,
+`mintapi` does not.
+Compatability with this method will be added in a later version.
+
+While you may disable MFA altogether (with some caveats),
+doing is not recommended.
+Not only will it decrease your account security,
+but Mint will sometimes **still email you a second factor code**.
+So, for the least fragility, enable MFA.
+
+The `mfa_method` parameter is only required if your login flow presents you with the option
+to select which Multifactor Authentication Method you wish to use,
+typically as a result of your account configured to accept different methods.
+Prior to v2.0, `mfa_method` is always required.
+
+- #### TOTP
+Set `mfa_method` to `soft-token`.
```

Comment:
This is in apparent contradiction to the previous paragraph, however, I was only able to get TOTP to work by setting the `mfa_method` to `soft-token`, so I think the previous paragraph should be modified. (Longer discussion in #484)

Reply:
@hudcap maybe it could be modified to state that it is best practice to try without mfa_method first, then specify one if it doesn't work, and open a bug report? Let me know if you have thoughts on this specific case @burkematthew

Reply:
I think there's a difference in application. By my understanding, setting `mfa_method` is only required past v2.0 for automated deployment, but not where `headless=False` during setup and/or troubleshooting. It's only required for automation because you don't want to sit there and manually enter the TOTP. (If this is wrong, please correct me.)

I'm not sure the best way to go about wording that in the README. If we expect most users to want automated deployment, we should default to simply saying that `mfa_method` is required. If we want to make the first-time startup instructions quicker/easier, I vote for leaving it marked as optional with a note along the lines of "required for automated use".

thoughts?

Reply:
We enhanced `mintapi` so that it detects which mfa method is requested by Mint.  It's been awhile since I've looked at it and have frankly never used a soft token.  So I can't remember _why_ it's necessary to specify.  The wording should basically indicate that unless you're using a `soft_token`, don't specify the `mfa_method`.  That being said, it's possible there's some edge casing and cleanup that could be done in the `soft_token` scenario so that it's a more uniform experience around MFA.

FWIW, using the email mfa_method allows for automation since you can use IMAP to connect with your inbox, which is how I've automated my implementation.